examples/js/loaders/GLTF2Loader.js

```diff
@@ -1430,16 +1430,12 @@ THREE.GLTF2Loader = ( function () {
 
 							_texture.type = texture.type !== undefined ? WEBGL_TEXTURE_DATATYPES[ texture.type ] : THREE.UnsignedByteType;
 
-							if ( texture.sampler !== undefined ) {
+							var sampler = texture.sampler === undefined ? {} : json.samplers[ texture.sampler ];
```

Comment:
`json.samplers` is optional.
So, how about this?

    var samplers = json.samplers || {};
    var sampler = samplers[ texture.sampler ] || {};